### ISSUE TITLE
Remove Do Not Track preference

### ISSUE THREAD

**Issue by arthuredelstein** [brave/brave-browser#32371]
## Description 

We are proposing to remove the Do Not Track preference from Brave Browser, because:
- it is obsolete: https://github.com/w3c/dnt/blob/master/README.md
- opting in to DNT adds fingerprinting entropy

**Comment by kimg45** [comment]
Yes please, other browsers like Safari already don't have an option in the UI for it.
